act-rules-format.md

```diff
@@ -497,6 +501,66 @@ Background (optional) {#background}
+
+There are three levels of consistency; complete, partial, and minimal. Consistency levels are defined in the following way. The consistency of a set of implementations procedures is the first level in the following list that matches the definition.
+
+1. <dfn>Complete</dfn> implementations consider all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `failed` test cases are reported as `failed`, or `cantTell` by at least one procedure; and
+    - All `passed` and `inapplicable` test cases are reported as `passed`, or `inapplicable`, or `cantTell` by all procedures; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` by all procedures.
+
+    Additionally, the set of [=implementation procedures=] reports all `failed` test cases as [=not satisfied=] for each [=accessibility requirement=] in the [requirements mapping](#accessibility-requirements-mapping), and for no other accessibility requirement in any [=accessibility requirements document=] in the requirements mapping.
+
+2. <dfn>Partial</dfn> implementations consider some, but not all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `passed` and `inapplicable` test cases are reported as `passed`, `inapplicable`, `cantTell`, or `untested`; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` or `untested`.
+
+3. <dfn>Minimal</dfn> implementations consider some or all aspects of a rule's applicability, without considering its expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `passed` and `inapplicable` test cases are reported as `passed`, `inapplicable`, `cantTell`, or `untested`; and
+    - `inapplicable` test cases are not all reported as `cantTell` or `untested`.
+
+
+### Coverage ### {#impl-coverage}
+
+Not all implementations are able to test all aspects of an ACT Rule. Particularly in automation, it is common for edge cases to be reported as `cantTell`. Coverage is a measure of how much of an ACT Rule is tested by an implementation. Similar to [consistency](#impl-consistency), the coverage of an ACT Rule is determined for a set of [=implementation procedures=]. Coverage is expressed using two numbers; total coverage, and automated coverage.
+
+The <dfn>total coverage</dfn> is expressed as a fraction where the denominator is the number of test cases, and the numerator is the number of test cases for which there was at least one correct outcome. `failed` outcomes are correct if it is for a `failed` test case. `passed` and `inapplicable` outcomes are correct if they are for a `passed` or `inapplicable` test case. Other outcomes are not considered correct.
```

Comment:
I'm getting a bit caught up on the phrase "for which there was at least one correct outcome." Is there not just one outcome submitted per test case?

Reply:
Looking at the example below, I see this is the scenario where an automated tool might say `cantTell` and then a manual review could say `pass` or `fail`. However, I'm still wondering if we should be more specific. It seems weird to me that an automated tool could say `fail` and then the manual review could say `pass` and overrule it. I would want to have proof that the tool and manual methodology correctly handle this scenario (e.g., does the tool notify the manual tester or does the manual tester have to review all of the outputs of the automatic tool?).

Reply:
Not sure what to do with this comment. Lets discuss on call.